src/commands/QuarantineMediaCommand.ts

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { MXCUrl } from "@vector-im/matrix-bot-sdk";
+import { Mjolnir } from "../Mjolnir";
+
+// !mjolnir quarantine-media <user ID> [room alias] [limit]
+// !mjolnir quarantine-media <server> [room alias] [limit]
+// !mjolnir quarantine-media <room ID> [limit]
+// !mjolnir quarantine-media <mxc-url>
+export async function execRedactCommand(roomId: string, event: any, mjolnir: Mjolnir, parts: string[]) {
+    const target = parts[2];
+
+    let targetRoom: string | null = null;
```

Comment:
I think we should do a check somewhere in here to ascertain if the bot is an admin or not and raise an error in the moderation room if not, as the quarantine command will fail in that case

Reply:
Done.